### ISSUE TITLE
MultiViews enabled by default. Should it be disabled?

### ISSUE THREAD

**Issue by rubenvarela** [lando/lando#3139]
**Please explain your request and its importance in the form of a user story.**

**Please provide some detail on how lando can help solve this problem**

**Is your feature request related to a problem? Please describe.**

On #2952 I documented an issue where we had some weird behavior where content that was supposed to be served wasn't and instead errors where being displayed. Looks like it's because Apache has MultiViews enabled. 

I mainly use the Drupal and Acquia recipes. I stumbled upon the behavior on there when I was testing something locally on Lando.

On the change record, https://www.drupal.org/node/2608772, it says, 

>If you have MultiViews enabled by default on your server, you should disable it with Options -MultiViews in the Apache configuration or in Drupal's root .htaccess file.

**Describe the solution you'd like**
Disabling MultiViews (and making sure FollowSymLinks or SymLinksIfOwnerMatch are enabled based on that change record)

Not sure if on the LAMP recipe, but at least match providers on their setups. On Acquia, apache configuration is locked down, but testing on their old DevDesktop, MultiViews is disabled and the error can't be recreated there.

**Describe alternatives you've considered**
Patching htaccess. I don't want to patch that for all sites when it's an environment issue on the platform (at least on the Acquia).

**Comment by rhelm** [comment]
I just ran into an odd bug that turned out to be an instance of this one. Specifically, I could not access content of a certain type in my local Lando Acquia plugin installation. I found a workaround, but I would like to hear whether there is a better solution, and whether it's a bug that the Acquia Lando plugin should address.
The reason I could not access content of the specific type was that the path alias for content of that type was of the form /update/*. The standard .htaccess in docroot was trying to serve update.php in response to requests to those paths, per the following rules:
```
  # Pass all requests not referring directly to files in the filesystem to
  # index.php.
  RewriteCond %{REQUEST_FILENAME} !-f
  RewriteCond %{REQUEST_FILENAME} !-d
  RewriteCond %{REQUEST_URI} !=/favicon.ico
  RewriteRule ^ index.php [L]
```
My workaround solution was to add the following code to the .htaccess file:
```
# Don't partial match file names when trying to find files for a URL
Options -MultiViews
```
Two questions:
1. What is the best way to introduce that directive into my local-only Apache config? I don't want to change the .htaccess in the Acquia cloud, which is working fine.
2. Is this something that needs to be fixed in the Lando Acquia plugin? It seems like the plugin's default setting for MultiViews does not match Acquia's cloud.
Thanks,
Rob

**Comment by reynoldsalec** [comment]
This should be resolved as of PHP v0.10.0, which will be available in Lando releases going forward. If you want to try installing the latest PHP version, you can manually install it using the "Docker" instructions: https://docs.lando.dev/php/#custom-installation
